src/Microsoft.Framework.PropertyHelper.Internal/PropertyHelper.cs

```diff
@@ -16,13 +16,19 @@ internal class PropertyHelper
         private delegate TValue ByRefFunc<TDeclaringType, TValue>(ref TDeclaringType arg);
 
         private static readonly MethodInfo CallPropertyGetterOpenGenericMethod =
-            typeof(PropertyHelper).GetTypeInfo().GetDeclaredMethod("CallPropertyGetter");
+            typeof(PropertyHelper).GetTypeInfo().GetDeclaredMethod(nameof(CallPropertyGetter));
```

Comment:
I did that purely for the PranavPoints. Saving up to buy a boombox. Any other feedback?